src/gui/fields/TakePoint.tsx

```diff
@@ -21,27 +21,69 @@
 import React from 'react';
 import {FieldProps} from 'formik';
 import Button, {ButtonProps} from '@material-ui/core/Button';
-import {Plugins} from '@capacitor/core';
+import {Plugins, GeolocationPosition} from '@capacitor/core';
+
+import {FAIMSPosition} from '../../datamodel/geo';
 
 const {Geolocation} = Plugins;
 
+function capacitor_coordindates_to_faims_pos(
+  coordinates: GeolocationPosition
+): FAIMSPosition {
+  const position = coordinates.coords;
+  const timestamp = coordinates.timestamp;
+  return {
+    type: 'Feature',
+    properties: {
+      timestamp: timestamp,
+      altitude: position.altitude ?? null,
+      speed: position.speed ?? null,
+      heading: position.heading ?? null,
+      accuracy: position.accuracy,
+      altitude_accuracy: position.altitudeAccuracy ?? null,
+    },
+    geometry: {
+      type: 'Point',
+      coordinates: [position.latitude, position.longitude],
```

Comment:
https://datatracker.ietf.org/doc/html/rfc7946#appendix-A.1 It's a trap (that I keep falling in. Order is long/lat (x,y)

Reply:
Fixed